requirements/quality.txt

```diff
@@ -240,8 +235,6 @@ rfc3986==2.0.0
     # via twine
 rich==12.6.0
     # via twine
-secretstorage==3.3.3
-    # via keyring
```

Comment:
@rgraber: Is `keyring` the Mac requirement? I guess someone else had committed this? I followed your instructions, and this is getting removed.

Reply:
It's a dependency when resolved on Linux: https://github.com/jaraco/keyring/blob/d706a01e13c75d89699d4244cea8baf36d78deee/setup.cfg#L23

Reply:
So is the solution to run the modified `make upgrade` in a container?

Reply:
I suspect so. I'm not sure what the least hacky way is of doing that.

Reply:
I found I don't really know what you mean, and if someone wants to write up simple instructions, that would be welcome. I did it Becca's way on my Mac and then removed any changes that weren't related. I noted in a new comment something odd that is happening, but I still think this is ready for approval.